src/components/map/layers/NodeLayer.tsx

```diff
@@ -30,9 +33,13 @@ enum color {
     STOP_BORDER_COLOR_SELECTED = '#007ac9',
     STOP_FILL_COLOR = '#FFF',
     STOP_FILL_COLOR_SELECTED = '#007ac9',
+    UNKNOWN_BORDER_COLOR = '#ff0000',
+    UNKNOWN_BORDER_COLOR_SELECTED = '#ff0000',
+    UNKNOWN_FILL_COLOR = '#000',
+    UNKNOWN_FILL_COLOR_SELECTED = '#000',
```

Comment:
If nodeType is unknown, should it fillcolor be white and border color red so that it would be more visible?